data/dataset/snowflake_example_test_dataset.yml

```diff
@@ -0,0 +1,229 @@
+dataset:
+  - fides_key: snowflake_example_test_dataset
+    name: Snowflake Example Test Dataset
```

Comment:
As we're thinking about test fixtures for these different datasources, it would be nice to create some interdependency between them, like how the mongo example test dataset relies on a field in the postgres example test dataset, I don't think our datasets should all be completely independent datastores.

Reply:
It would be nice to have the postman collection be able to make requests against our fidesops snowflake db - but the way i understand it, we wipe and repopulate it from these unit tests?

Reply:
> we wipe and repopulate it from these unit tests?

Not quite — we're only wiping and repopulating the data we use in erasures. Everything else is part of a copy of the dataset Atlas uses for Snowflake. There are some frustrating minor differences here, for example, in Snowflake the collection is `order` and in Postgres it's `orders` plural.

I'll factor this into the testing design for sure! Thanks